src/refactor/index.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+
+// return on each Click the value of Square
+function Square(props) {
+  return (
+    <button className="square" onClick={props.onClick}>
+      {props.value}
+    </button>
+  )
+}
+// Describe Board Game + set boolean to setup user 
+class Board extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      squares: Array(9).fill(null),
+      xIsNext: true,
+    };
+  }
+
+  // handle which turn is it
+  handleClick(i) {
+    const squares = this.state.squares.slice();
```

Comment:
Why you are slicing here?

Reply:
I thought only with slice it will be able to understand the squares Array.
any way you right and its work without it 🥇